platform/broadcom/sonic-platform-modules-cel/dx010/modules/Makefile

```diff
@@ -1,6 +1 @@
 obj-m := dx010_cpld.o mc24lc64t.o emc2305.o dx010_wdt.o leds-dx010.o lm75.o
```

Comment:
you still have lm75.o here?

Reply:
can you make sure your dx010 module can compile as it is?

Reply:
Yes, it should be removed. I've fixed